src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -769,8 +805,8 @@ private void cleanupNode(boolean willShutdown) {
     }
 
     private boolean keepVideoAndLogs() {
-        return !keepOnlyFailedTests || TestInformation.TestStatus.FAILED.equals(testInformation.getTestStatus())
-                || TestInformation.TestStatus.TIMEOUT.equals(testInformation.getTestStatus());
+        return isVideoRecordingEnabled() && (!keepOnlyFailedTests || TestInformation.TestStatus.FAILED.equals(testInformation.getTestStatus())
```

Comment:
Why do we need to add this condition now?

Reply:
Looking at this again, this doesn't seem necessary. I can change it back. I think I added to left side so that it doesn't have to evaluate the right side if it doesn't have to.

Reply:
OK, then let's leave it the way it was 🤝